openassessment/workflow/models.py

```diff
@@ -346,12 +355,61 @@ def _get_steps(self):
             # If no steps exist for this AssessmentWorkflow, assume
             # peer -> self for backwards compatibility
             self.steps.add(
-                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=0),
-                AssessmentWorkflowStep(name=self.STATUS.self, order_num=1)
+                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=1),
+                AssessmentWorkflowStep(name=self.STATUS.self, order_num=2)
             )
             steps = list(self.steps.all())
+
+        # A staff step will always be available, to allow for staff overrides
```

Comment:
I think that makes sense. I guess the other concern is when we're rendering this on the front end. If the list of steps for a problem includes the staff step all the time, does this mean that it will always get rendered? Or do we want to put in an exception here?